com.microsoft.mrtk.input/Visualizers/RiggedHandVisualizer/RiggedHandMeshVisualizer.cs

```diff
@@ -0,0 +1,295 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+
+using Microsoft.MixedReality.Toolkit.Subsystems;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.XR;
+using UnityEngine.XR.Interaction.Toolkit;
+
+using InfoType = Microsoft.MixedReality.Toolkit.ShowInfoIfAttribute.InfoType;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// Hand visualizer that uses a rigged mesh/armature to render high-quality hand meshes.
+    /// Not recommended for AR platforms like HoloLens, both for performance and design reasons. 
+    /// </summary>
+    /// <remarks>
+    /// For augmented reality platforms such as HoloLens, we recommend not using any hand visualizations,
+    /// as the conflict between the user's real hand and the slightly delayed holographic visualization
+    /// can be more distracting than it's worth. However, for opaque platforms, this is a great solution.
+    /// </remarks>
+    [AddComponentMenu("MRTK/Input/Visualizers/Rigged Hand Mesh Visualizer")]
+    public class RiggedHandMeshVisualizer : MonoBehaviour
+    {
+        [SerializeField]
+        [Tooltip("The XRNode on which this hand is located.")]
+        private XRNode handNode = XRNode.LeftHand;
+
+        /// <summary> The XRNode on which this hand is located. </summary>
+        public XRNode HandNode { get => handNode; set => handNode = value; }
+
+        [SerializeField]
+        [Tooltip("When true, this visualizer will render rigged hands even on XR devices with transparent displays.")]
```

Comment:
```suggestion
        [Tooltip("When true, this visualizer will render rigged hands on XR devices with transparent displays. The visualizer always render rigged hands on devices with opaque displays.")]
```

Reply:
Rewrote the tooltip to be more explicit.